copilot/fastapi/app/kubernetes_api.py

```
@@ -0,0 +1,110 @@
+            response = await client.request(
+                method=request.method,
+                url=target_url,
+                headers=headers,
+                content=body,
+                follow_redirects=True
+            )
+            
+            # Return the raw response
+            return response.json()
+        except httpx.HTTPError as e:
+            raise HTTPException(status_code=503, detail=f"Kubernetes API unavailable: {str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)